app/models/grda_warehouse/hud/client.rb

```diff
@@ -496,6 +496,14 @@ def hmis_source_visible_by?(user)
       where(Arel.sql('1').in(races.map { |race| c_t[race] }))
     end
 
+    # a scope where no race was chosen
+    # potentially this should include RaceNone: [8, 9, 99], but if all others are 0 or 99, then
+    # we really don't know the race
+    scope :with_race_none, -> do
+      not_race = [0, 99]
+      where(AmIndAKNative: not_race, Asian: not_race, BlackAfAmerican: not_race, NativeHIPacific: not_race, White: not_race)
```

Comment:
Do we need `nil in here`?
```suggestion
      query = []
      [:AmIndAKNative, :Asian, :BlackAfAmerican, :NativeHIPacific, :White].each do |key|
        part = c_t[key].in([0, 99].or(c_t[key].eq(nil)
        query = query.present? ? query.and(part) : part
      end
      where(query)
```

Reply:
These aren't null-able per the spec